### DISCUSSION TITLE
Is there a comprehensive way of using Turborepo while developing?

### DISCUSSION THREAD

**Discussion by pedropmedina** [vercel/turborepo#3229] [Help] [answered]
I recently decided to migrate to Turborepo as I like the ways it handles monorepo more than NX. However I'm having issues getting it to update the parent project on changes to its dependencies packages. For instance if I have a project with 3 packages (pkg1, pgk2, pgk3) where pkg2 and pkg3 are dependencies of pkg1. I'm running the dev with tsup --watch and when I make changes to pkg2 or pkg3, pkg1 won't update with the changes.

Is there a native way of handling this or am I missing something. Here's my turbo.json file

```json
{
  "$schema": "https://turbo.build/schema.json",
  "globalEnv": ["NODE_ENV", "REDIS_HOST", "REDIS_PORT"],
  "pipeline": {
    "dev": {
      "dependsOn": ["^build"],
      "cache": false
    },
    "build": {
      "dependsOn": ["^build", "db:generate"],
      "outputs": ["dist/**", ".next/**"]
    },
    "execute": {
      "cache": false
    },
    "lint": {
      "outputs": []
    },
    "clean": {
      "cache": false
    }
  }
}

```

**Comment by mehulkar** [comment, marked as answer]
> update the parent project on changes to its dependencies packages

we don't support this behavior at the moment. But you can use the `turbo run dev --parallel` flag to run all dev servers independently. Changes in one package will not trigger a reload in another package, at the moment. 

In Turbo 1.7, we added the [`persistent` config](https://turbo.build/blog/turbo-1-7-0) to _prevent_ this behavior--configuring your dev servers from depending on each other.
